client/widgets/CArtifactsOfHeroAltar.cpp

```diff
@@ -22,6 +22,8 @@
 CArtifactsOfHeroAltar::CArtifactsOfHeroAltar(const Point & position)
 {
 	init(position, std::bind(&CArtifactsOfHeroBase::scrollBackpack, this, _1));
+	setClickPrassedArtPlacesCallback(std::bind(&CArtifactsOfHeroBase::clickPrassedArtPlace, this, _1, _2));
```

Comment:
You might want to do search and replace Prassed -> Pressed
```suggestion
	setClickPressedArtPlacesCallback(std::bind(&CArtifactsOfHeroBase::clickPressedArtPlace, this, _1, _2));
```